src/main/telemetry/hott.c

```diff
@@ -415,8 +444,88 @@ static void hottPrepareMessages(void) {
 #endif
 }
 
+#if defined (USE_HOTT_TEXTMODE) && defined (USE_CMS)
+static void hottTextmodeStart()
+{
+    // Increase menu speed
+    cfTaskInfo_t taskInfo;
+    getTaskInfo(TASK_TELEMETRY, &taskInfo);
+    telemetryPeriod = taskInfo.desiredPeriod;
+    rescheduleTask(TASK_TELEMETRY, TASK_PERIOD_HZ(1000));
```

Comment:
maybe just use macro to define task rate (as for HOTT_TEXTMODE_RX_SCHEDULE)

Reply:
Oops, overlook.

Reply:
Sorry, bad wording.
do not use telemetryPeriod, but use hardcoded values HOTT_TEXTMODE_RX_SCHEDULE / HOTT_RX_SCHEDULE